docker-compose.test.yaml

```diff
@@ -5,6 +5,36 @@ version: '3.4'
 
 services:
   ffc-demo-user-service:

```

Comment:
The service name should be different to the one defined in `docker-compose.yaml`. It doesn't conflict at the moment because there's no explicit `container_name` declaration here, so Docker concatenates the folder name, service name and a random string into a very long container name (e.g. `ffc-demo-user-service_ffc-demo-user-service_run_ec6f9f34629d`).

Might be nice to set an explicit container name in line with the others, such as `ffc-demo-user-test` and change the service name to match.